### ISSUE TITLE
Requesting Functionality to add multiple systems or users by a script   

### ISSUE THREAD

**Issue by sudeepgalgali** [skavanagh/KeyBox#180]
Hey,
I am thrilled by the functionality this application provides. However to fit my use case. I require a way to populate the database via a script. In our organization we have hundreds of systems and it is not feasible to add systems manually. The same applies to users as well. Can anything be done in this regard?

On an another note what is the default password for the user "keybox" to access the h2 database?

**Comment by skavanagh** [comment]
> Can anything be done in this regard?

Maybe!  I'd prefer the script connect to the DB directly, rather than having a REST API that the server exposes.  Authentication becomes problematic with a REST API and not sure how something like that would work with 2FA (and I don't really want to turn 2FA off for the API).

> On an another note what is the default password for the user "keybox" to access the h2 database?

I've been doing work around this (haven't released yet), and you can set the default password for the keybox user in the KeyBoxConfig.properties prior to starting the application for the first time.  

https://github.com/skavanagh/KeyBox/blob/master/src/main/resources/KeyBoxConfig.properties#L48-L49

If the password is in plain-text the value is encrypted, and updated in the properties file once the application is started. The encryption key for this is generated on startup too, and keystore is created for it.

**Comment by sudeepgalgali** [comment]
> https://github.com/skavanagh/KeyBox/blob/master/src/main/resources/KeyBoxConfig.properties#L48-L49

I dont see that in an older source code, is it a new addition?

**Comment by skavanagh** [comment]
Yes, it's a new addition. And I haven't released it yet (maybe have some
time this weekend).
